frontends/concrete-python/concrete/fhe/__init__.py

```diff
@@ -193,3 +193,26 @@
 from .version import __version__
 
 # pylint: enable=import-error,no-name-in-module
+
+import warnings
+from cpuinfo import get_cpu_info
+
+def _is_cpu_compatible() -> bool:
+    """Check if CPU can run Concrete
```

Comment:
maybe 
CPU has AES instruction
they can still run compilation ?

Reply:
Still not sure to check for aes only

Reply:
aes is not actually an issue I guess, as we fallback to a software implementation